src/domain/balances/entities/balance.entity.ts

```diff
@@ -1,12 +1,12 @@
 import { BalanceToken } from '@/domain/balances/entities/balance.token.entity';
 
-interface NativeBalance {
+export interface NativeBalance {
```

Comment:
I had to export these two interfaces to avoid transpiler errors by explicitly state the return types in `valk-balances-api.service.ts`.